superbench/config/default.yaml

```diff
@@ -3,7 +3,7 @@ version: v0.9
 superbench:
   enable: null
   monitor:
-    enable: true
+    enable: false
```

Comment:
Why change this?

Reply:
Since lot's of UT will use this default config file and create monitors even don't meant to test it. Just want to accelate the UT process.
I have reverted this change.,